SDWebImage/SDWebImageGIFCoder.m

```diff
@@ -72,13 +72,15 @@ - (UIImage *)decodedImageWithData:(NSData *)data {
             [frames addObject:frame];
         }
         
-        NSUInteger loopCount = 0;
+        NSUInteger loopCount = 1;
         NSDictionary *imageProperties = (__bridge_transfer NSDictionary *)CGImageSourceCopyProperties(source, nil);
         NSDictionary *gifProperties = [imageProperties valueForKey:(__bridge_transfer NSString *)kCGImagePropertyGIFDictionary];
         if (gifProperties) {
             NSNumber *gifLoopCount = [gifProperties valueForKey:(__bridge_transfer NSString *)kCGImagePropertyGIFLoopCount];
-            if (gifLoopCount != nil) {
-                loopCount = gifLoopCount.unsignedIntegerValue;
+            if (gifLoopCount != nil && gifLoopCount.unsignedIntegerValue == 0) {
+                loopCount = 0;
+            } else {
+                loopCount = gifLoopCount.unsignedIntegerValue + 1;
```

Comment:
@LululuSir This If has no loopCount info, loop count would be 1 seems adoptable. But what about these change ? Should we add loop count by 1 ?